livingdoc-fixture-converter/src/main/java/org/livingdoc/fixture/converter/number/ByteConverter.kt

```diff
@@ -0,0 +1,6 @@
+package org.livingdoc.fixture.converter.number
+
+
+open class ByteConverter : AbstractNumberConverter<Byte>() {
```

Comment:
I'll Implement `BigInteger` and `BigDecimal` in a separate PR. Additionally we should implement a check for all our `Number` converters regarding their valid range. This should prevent range overflows. But not in this PR.